values/gitea/gitea.gotmpl

```diff
@@ -0,0 +1,51 @@
+{{- $v := .Environment.Values }}
+{{- $o := $v.oidc }}
+{{- $g := $v.charts | get "gitea" dict }}
+{{- $k := $v.charts | get "keycloak" dict }}
+{{- $hasKeycloak := $k | get "enabled" true }}
+{{- $realm := $k | get "realm" "master" }}
+{{- $keycloakIssuer := printf "https://keycloak.%s/realms/%s" $v.cluster.domain $realm }}
+
+{{- $giteadomain := printf "gitea.%s" $v.cluster.domain}}
+{{- $addIgnorePolicyAnnotation := readFile "../../helmfile.d/utils/addIgnorePolicyAnnotation.gotmpl" }}
+
+nameOverride: gitea
+fullnameOverride: gitea
+# clusterDomain: {{ $v.cluster.domain }}
+
+gitea:
+  podAnnotations: 
+    {{-  tpl $addIgnorePolicyAnnotation (dict "policies" (list "psp-allowed-users"))  | nindent 4 }}
+  admin:
+    username: otomi
+    password: {{ $g | get "admin.password" "bladibla" }}
+  config:
+    server:
+      DOMAIN: {{ $giteadomain }}
+      ROOT_URL: "https://%(DOMAIN)s/"
```

Comment:
why that trailing `s`? And if the default value of `ROOT_URL` is "https://%(DOMAIN)/" then we don't even have to set it

Reply:
Because that's how gitea parses ENV variables: `%(VAR_HERE)s`
https://docs.gitea.io/en-us/config-cheat-sheet/#server-server (See ROOT_URL)

Reply:
And we have to set it to override HTTP with HTTPS, and omit port 3000. We can't change the PROTOCOL, because otherwise it would complain about certificates. So override the ROOT_URL was necessary. As this variable builds the URL used throughout the app, to set all links to `https://gitea.dev.eks.otomi.cloud/`

Reply:
can we just use our own style here then? And not expose something nobody needs to know about?

so please make it:
```suggestion
      ROOT_URL: "https://{{ $giteadomain }}/"
```